hack/generator/pkg/astmodel/interface_implementer.go

```diff
@@ -46,31 +46,31 @@ func (i InterfaceImplementer) AsDeclarations(
 
 	var result []ast.Decl
 
-	// First interfaces must be ordered by name for deterministic output
-	var ifaceNames []TypeName
-	for ifaceName := range i.interfaces {
-		ifaceNames = append(ifaceNames, ifaceName)
+	// interfaces must be ordered by name for deterministic output
+	// (We sort them directly to skip future lookups)
+	var interfaces []*InterfaceImplementation
+	for _, intf := range i.interfaces {
+		interfaces = append(interfaces, intf)
 	}
 
-	sort.Slice(ifaceNames, func(i int, j int) bool {
-		return ifaceNames[i].name < ifaceNames[j].name
+	sort.Slice(interfaces, func(i int, j int) bool {
+		return interfaces[i].Name().name < interfaces[j].Name().Name()
```

Comment:
Probably best to be consistent on either side of the `<`.

Reply:
Fixed to directly access the member `name` on both sides of the ordering comparison.